File: android/src/main/kotlin/com/syncrhrohealth/bandpass_ble_connection/plugin/blestate/constant/BleStateMethod.kt
package com.syncrhrohealth.bandpass_ble_connection.plugin.blestate.constant

/** Make sure the method value matches the BleStateMethod on Flutter side */
enum class BleStateMethod(val value: String) {
    IS_BLE_ENABLED("isBleEnabled"),
    ENABLE_BLE("enableBle")
}

File: ios/Classes/Plugin/BleState/Constant/BleStateMethod.swift
//
//  BleStateMethod.swift
//  Runner
//
//  Created by MAC on 12/3/25.
//

import Foundation

/// Make sure the method value matches the `BleStateMethod` on the Flutter side
enum BleStateMethod: String {
    case isBleEnabled = "isBleEnabled"
    case enableBle = "enableBle"
}
